examples/wCCD/src/wCCD.tsx

```diff
@@ -1,13 +1,18 @@
 /* eslint-disable no-console */
```

Comment:
- Why is the disconnect button darker than others, instead of the standard green?
- When I disconnect my account with the browser wallet, I get to:
![Screenshot from 2022-12-13 14-41-55](https://user-images.githubusercontent.com/87635671/207340614-b0515aff-312a-47a2-bc0c-f42933dcfca8.png)
I guess the connect/disconnect buttons should reset? :thinking: 

- When I disconnect with wallet-connect in the mobile wallet, nothing happens on the dApp? :thinking:
(and vice versa?)
Should they not get disconnect events? (It might be that I have an outdated wallet) :thinking:

Reply:
The color was set to indicate that "Use Browser Wallet" was selected. This was before I changed it to be a disconnect button. I think there needs to be a visual indicator of which connection type is used, but am not sure if this is the best solution. Maybe if the button turned red and said "Disconnect Browser Wallet" would be better?

Reply:
I haven't seen this error other than the mobile wallet often not picking up that the dapp disconnected. Do you get anything useful in the console?

Reply:
I love the new user flow that the user has to actively disconnect before being able to switch to a new connection. Well done. I also agree that we should make the `Disconnect` button red and call it `Disconnect Browser Wallet/Disconnect Wallet Connect`.

When disconnecting in the browser wallet then this is something the front-end cannot pick up it seems. If we make the `wrap/unwrap` button grey in that case, the user would know that it needs to `disconnect` (only button left to click on dApp front-end) the connection on the dApp as well.
![Screenshot from 2022-12-14 11-45-38](https://user-images.githubusercontent.com/13039169/207564298-a7686db0-4749-46e3-96d5-e2cd683b2692.png)

Reply:
Updated styling of both disconnect and styling buttons as you suggested.